modules/hosted-control-planes-pause-reconciliation.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-pause-reconciliation_{context}"]
+= Pausing the reconciliation of a hosted cluster and hosted control plane
+
+If you are a cluster instance administrator, you can pause the reconciliation of a hosted cluster and hosted control plane. You might want to pause reconciliation when you back up and restore an etcd database or when you need to debug problems with a hosted cluster or hosted control plane. 
+
+.Procedure
+
+. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following example:
++
+[source,terminal]
+----
+PAUSED_UNTIL="2022-03-03T03:28:48Z"
+kubectl patch -n <hosted-clusters-namespace> hostedclusters/<hosted-cluster-name> -p '{"spec":{"pausedUntil":"'${PAUSED_UNTIL}'"}}' --type=merge
+----
++
+The value in the `pausedUntil` field can be a Boolean value, such as `PAUSED_UNTIL="true"` or an RFC339 timestamp, as shown in the example. If you specify an RFC339 date, the reconciliation is paused until that date. If you pass a Boolean value of `true`, reconciliation is paused until the field is removed from the `HostedCluster` resource.
```

Comment:
nit: `If you specify an RFC339 date` should date be timestamp just to be consistent with the previous usage?

Reply:
Yes. I'll change that. Good catch!